lux/vislib/altair/Histogram.py

```diff
@@ -55,38 +55,39 @@ def initialize_chart(self):
         markbar = x_range / plot_range * 12
```

Comment:
What's the point of this formula? The *12 looks suspicious to me.

Reply:
From my understanding, `markbar` is Lux's way of determining a consistent bin size across all histograms. Otherwise, Altair will estimate and provide their own bin size. In my recent commit, I change the multiplier to `10`, which yields output like so: 
![Screen Shot 2021-02-22 at 12 34 20 PM](https://user-images.githubusercontent.com/44735047/108767999-fa569380-750b-11eb-98a6-4aeadd6348d2.png)

When I remove the `markbar` constraint entirely, Altair will interpret the appropriate bin size automatically. This is the example yielded output:

![Screen Shot 2021-02-22 at 12 41 56 PM](https://user-images.githubusercontent.com/44735047/108768149-2b36c880-750c-11eb-9426-937a92d3ccb4.png)

The key difference is that in the latter input, the bin size is determined for each individual graph. In the former input, the bin size is set in a way such that bin width appears to be the same width across all graphs such that precisely 10 bars are displayed in each graph, giving a more consistent look. From an HCI perspective, I'm not in a position to comment on which one is more meaningful. Perhaps @domoritz you have some input that would help us make a decision on this. 

@dorisjlee, feel free to chime in and correct me if there's anything I misinterpreted.

Reply:
I think the bins Vega uses are a lot better since they fall on understandable steps but since you pre-aggregate the data, you would need to reimplement the logic in https://github.com/vega/vega/blob/ca27004f8c3ad8aa94f017fb5df6bf90748bda92/packages/vega-statistics/src/bin.js. 

It's probably best to do that later, though, and first merge this fix that sets bin start and end and passes the correct bin size.

12 is fine (as fine as 10). I just think the logic could be more sophisticated to compute bin start and step (using the logic linked above and maybe combined with e.g. Sturge's Rule).